block_designs.tex

```diff
@@ -388,6 +388,92 @@ \section{Hadamardove matice}
 Dokážte platnosť konštrukcie ku vete \ref{th:hm_diffset} pre druhý smer. 
 \end{exercise}
 
+\begin{theorem}
+\label{th:hm_paley}
+Ak platí, že $n = p^r + 1 \equiv 0 (\textrm{mod}\ 4)$ a zároveň $p$ je nepárne prvočíslo tak existuje Hadamardova matica rádu $n$ a takéto matice voláme Payleyho.
```

Comment:
Nepíšeš, čo je r. Ale ak je jasné, že to je prirodzené číslo, lebo očakávam, že v celých skriptách sú prirodzené a navyše nemôže byť 0 kvôli podmienke, tak to podľa mňa môže ostať aj tak.
Aké matice sa volajú Payleyho? Hocijaké rádu p^r + 1, ktorý dáva zvyšok 3 mod 4? Alebo len tie, ktoré sú skonštruované týmto spôsobom? Skús to tam nejako ujasniť. Ak by to mala byť prvá varianta, tak by som čakal, že definícia Payleyho matíc je samostatne pred vetou. Ak je to druhý prípad, tak by stačilo niečo ako: Matice skonštruované v tejto vete voláme Payleyho.

Reply:
Pokúsim sa upresniť.